mlir/lib/Conversion/TosaToLinalg/TosaToLinalg.cpp

```diff
@@ -563,7 +563,7 @@ static Value createLinalgBodyCalculationForElementwiseOp(
                    getElementTypeOrSelf(srcTy),
                    APInt::getSignedMaxValue(dstTy.getIntOrFloatBitWidth())
                            .getSExtValue() +
-                       1));
+                       1.0));
```

Comment:
nit: Suppose this now works due to the implicit conversion taking place.
Probably better to be more explicit in converting the above to double?